src/msix/pack/AppxPackageWriter.cpp

```diff
@@ -254,4 +238,30 @@ namespace MSIX {
         ThrowErrorIfNot(Error::InvalidParameter, result, "Invalid compression option.");
     }
 
+    // Common close functionality for pack and bundle.

```

Comment:
>// Common close functionality for pack and bundle. [](start = 4, length = 50)

same here, you can actually revert this file entirely and fix the build breaks

Reply:
Reverted this file entirely except for the change to the IsFootPrintFile method in ValidationHelper since that method checks for different files based on if it is a package or a bundle.